databricks/koalas/base.py

```diff
@@ -299,10 +301,25 @@ def is_monotonic(self):
 
         >>> ser.rename("a").to_frame().set_index("a").index.is_monotonic
         True
+
+        >>> ser = ks.Series([5, 4, 3, 2, 1], index=[1, 2, 3, 4, 5])
+        >>> ser.is_monotonic
+        False
+
+        >>> ser.index.is_monotonic
+        True
         """
+        from databricks.koalas.frame import DataFrame
+        from databricks.koalas.series import _col
         col = self._scol
-        window = Window.orderBy(self._kdf._internal.index_scols).rowsBetween(-1, -1)
```

Comment:
I think the only fix needed is replace `self._kdf._internal.index_scols` to `monotonically_increasing_id()`. Other changes look not needed.

Reply:
@HyukjinKwon Thanks for reviewing..! ugh... it was such a simple problem if i took more carefully 😿